profiles/car.lua

```diff
@@ -606,6 +606,16 @@ function way_function (way, result)
     end
   end
 
+  -- add driveability penalty to service roads
+  if properties.weight_name == 'duration' and highway == "service" then
+    if result.duration > 0 or result.forward_speed > 0 then
+      result.forward_weight_per_meter = result.forward_speed / 100;
```

Comment:
The `weight_per_meter` name is slightly misleading, this represents a value analogous to `speed`, where higher values mean faster roads, lower values mean slower roads.

I definitely felt the vocabulary pain here, and would totally be open to finding different wording.